### DISCUSSION TITLE
Google Basic URL for Android

### DISCUSSION THREAD

**Discussion by RobinManoli** [InteractionDesignFoundation/add-event-to-calendar-docs#62] [General]
Hi,
I've noticed that using https://calendar.google.com/calendar/r/eventedit, with or without action=TEMPLATE, does not work on Android phones (tested a new Samsung and an old Huawei). A link that uses that URI only opens Google Calendar, but does not initiate the calendar event creation interface.

Switching to https://calendar.google.com/calendar/render makes it work.

**Comment by alies-dev** [comment]
Cannot verify the Android half without a device, so leaving this open, but the desktop half is now confirmed and the document has been updated accordingly.

On the web, `https://calendar.google.com/calendar/render?action=TEMPLATE&...` is a thin entry point: it answers with a redirect to `https://calendar.google.com/calendar/u/0/r/eventedit` and carries the whole query string across unchanged. Both forms therefore reach the same editor with the same parameters in a browser, which matches the report that only the difference in behaviour shows up on Android.

Given that, there is no cost to preferring `render` everywhere, and #59 now says so in the Basic URL section:

> Prefer `render` when you do not control the device the link is opened on. It is the form Google itself hands out, and it is reported to be the only one that reaches the event editor on Android, where the `eventedit` path opens the Google Calendar app without starting event creation.

If anyone can retest `render` versus `eventedit` on a current Android build, that would settle whether the `eventedit` intent filter is still the problem or whether it has since been fixed.
